dev-qt/qt-creator/qt-creator-9999.ebuild

```diff
@@ -59,7 +59,18 @@ CDEPEND="
 	>=dev-qt/qtwidgets-${QT_PV}
 	>=dev-qt/qtx11extras-${QT_PV}
 	>=dev-qt/qtxml-${QT_PV}
-	clang? ( sys-devel/clang:8= )
+	kde-frameworks/syntax-highlighting:5=
+	clang? (
+		|| (
+			( sys-devel/clang:10
+				dev-libs/libclangformat-ide:10 )
+			( sys-devel/clang:9
+				dev-libs/libclangformat-ide:9 )
+			( sys-devel/clang:8
+				dev-libs/libclangformat-ide:8 )
```

Comment:
llvm proj have already opened a tracker for LLVM-8 removal, I think we can slim down here a bit.